charmcraft/extensions/__init__.py

```diff
@@ -18,7 +18,7 @@
 
 from charmcraft.extensions._utils import apply_extensions
 from charmcraft.extensions.extension import Extension
-from charmcraft.extensions.gunicorn import DjangoFramework, FlaskFramework
+from charmcraft.extensions.twelvefactor import DjangoFramework, FlaskFramework, GoFramework
```

Comment:
let's change it to `app` with a backup option of `app_framework`

Reply:
Renamed to app